build-output-api/image-optimization/README.md

```diff
@@ -2,6 +2,8 @@
 
 ### Build Output API
 
+#### [Demo](https://build-output-api-image-optimization.vercel.sh)
```

Comment:
Can we add `width` and `height` to the `<img>` so it doesn't have CLS?

Reply:
Added